### ISSUE TITLE
Error in HTML report

### ISSUE THREAD

**Issue by gitPrasun** [archimatetool/archi#450]
I see a usual problem with reporting to HTML. While I inspect it in chrome , it always says the below error:

`Uncaught TypeError: Cannot read property 'replace' of undefined
    at HTMLDocument.<anonymous> (frame.js:127)
    at j (jquery-1.11.2.min.js:2)
    at Object.fireWith [as resolveWith] (jquery-1.11.2.min.js:2)
    at Function.ready (jquery-1.11.2.min.js:2)
    at HTMLDocument.J (jquery-1.11.2.min.js:2)
(anonymous) @ frame.js:127
j @ jquery-1.11.2.min.js:2
fireWith @ jquery-1.11.2.min.js:2
ready @ jquery-1.11.2.min.js:2
J @ jquery-1.11.2.min.js:2`
![image](https://user-images.githubusercontent.com/33220251/54186506-dbd86880-44ab-11e9-8601-5b9a41d24fb6.png)

Though it does not cause any visual problem at the moment - is it normal ? or is there there any setting that I miss.

### Version of Archi, Operating System

**Archi version 4.3.3**

### Expected Behaviour

The HTML report is expected to have free of errors

### Actual Behaviour

The inspect of the generated HTML has the afforesaid error.

### Steps to Reproduce the Behaviour
1. generate HTML report from File->Report-> html.. option
2. Open the report in browser
3. Inspect the html ( in chrrome)

**Comment by jbsarrodie** [comment]
This is more a warning than an error. This part of the script is common to all element-related sub pages and is meant to transform any URL or email in description field to a real link that opens in a new browser window or tab.

This script is also included in the "empty-element.html" page which shows the default message "Select an element in the model tree or in a view to show its details here", but in this context there is no description field to transform, thus the message.

I guess this can be easily fixed. I'll have a look the next time I'll work on HTML report.

**Comment by Phillipus** [comment]
I took a look at it. If we check for undefined it seems to do the trick:

```
    if(typeof $('#doctgt').html() !== "undefined") {
		$('#doctgt').html($('#doctgt').html()
			.replace(/(\w+:\/\/\S+)/g, '<a href="$1" target="_blank">$1</a>')
			.replace(/mailto:(\S+)/g, '<a href="mailto:$1">$1</a>')
		);
    }
```

@jbsarrodie Would that be OK?

**Comment by Phillipus** [comment]
Tested that in new branch and seems to be OK.

**Comment by jbsarrodie** [comment]
That should be ok. TBH I was thinking about simply adding two hidden spans with needed ids.

**Comment by Phillipus** [comment]
Fixed in Archi 4.4 beta
